tests/integration/processing/Lexicon.py

```diff
@@ -229,6 +229,48 @@ def __init__(self, cases, preparation_lines=''):
                          key='a', expected='helloworld'))
         ]
     ),
+    TestSuite(
+        preparation_lines='list = ["hello", "world"]\n'
+                          'dict = {"hello": "world"}\n'
+                          'file write path: "/tmp.txt" content: "hello"\n'
+                          'bytes = file read path: "/tmp.txt" raw: true',
+        cases=[
+            TestCase(prepend='a = "foo is {true}"',
```

Comment:
Why not just 
```suggestion
            TestCase(prepend='a = "{true}"',
```

or `true as string`